### DISCUSSION TITLE
Japanese TV Channels

### DISCUSSION THREAD

**Discussion by kin37ik** [iptv-org/community#1044] [Q&A]
Hello all

I did have working links to Nippon TV, TBS, TV Asahi, Fuji TV and TV Tokyo but they have recently gone offline and does not look like they will be coming back, i got these TV channels links from September 2023 in a discussion among here somewhere and haven't found any alternatvies recently.

Is anyone else aware of any alternatives that someone could point me too?

**Comment by Animenosekai** [comment]
The streams provided in [jp.m3u](https://github.com/iptv-org/iptv/blob/master/streams/jp.m3u) and [jp_primehome.m3u](https://github.com/iptv-org/iptv/blob/master/streams/jp_primehome.m3u) still seem to work

**Reply by kin37ik** [reply]
they do not work for me at all, they used to work and then seem to have gone offline or unreachable.

I live in Australia so i do not know if there is any geoblocking happening.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
